api/v1/zz_generated.deepcopy.go

```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```

Comment:
Happens when running `make generate` and `make manifests`. Probably depends on the operator-sdk version.